src/components/camara/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import camara from '../assets/camara.png';
+import styles from './styles.module.scss';
+
+class Camara extends React.Component {
+    render() {
+        return(
+            <div>
+                    <div className={styles.cam}>
+                        <img src={camara} alt='camara'/>
+                    </div>
+                
+            </div>
```

Comment:
Prestar atención a los espaciados, indentación y espacios de más (y de menos)
```suggestion
        return (
            <div>
                <div className={styles.cam}>
                    <img src={camara} alt='camara' />
                </div>
            </div>
```

También, ¿es necesario envolver la imagen en dos `div`s?

Reply:
Un div encierra la imagen, el otro div encierra la topbar, ya esta corregido !